File: Atividades/Reforço de Lógica/PHP/EX04.php
<?php 
// Pedindo a temperatura:
$temperatura = readline("Digite a temperatura atual em graus celsius: ");

// Se a temperatura for maior que 27, está quente:
if ($temperatura > 27){
    echo "\033[31mEstá quente.\033[m";
// Se for menor que 15, está frio:
} elseif($temperatura < 15){
    echo "\033[34mEstá frio.\033[m";
// Se estiver entre 15 e 17, está agradável:
} else{
    echo "\033[32mEstá agradável.\033[m";
}
?>

File: Atividades/Reforço de Lógica/Python/EX04.py
# ------------------- Verifica a Temperatura ------------------- #

temperatura = int(input("Digite a temperatura em graus celsius: ")) # Pedindo a temperatura

if temperatura > 27: # Se a temperatura for maior que 27 graus, está quente:
    print("\033[31mEstá quente.\033[m")

elif temperatura < 15: # Se a temperatura for menor que 15 graus, está frio:
    print("\033[34mEstá frio.\033[m")

else: # Se não estiver nem maior que 27 graus e nem menor que 15, está agradável:
    print("\033[32mEstá agradável.\033[m")